pulp_rpm/app/models/repository.py

```diff
@@ -16,6 +16,13 @@
 )
 from pulpcore.plugin.repo_version_utils import remove_duplicates
 
+from pulp_rpm.app.models import (
```

Comment:
I think we are trying to have one object per line and in alphabetical order. Up to you.